### ISSUE TITLE
23.07.18 (화) Issue

### ISSUE THREAD

**Issue by SaFD-00** [SaFD-00/2023_RCV_SUMMER_URP#16]
# <공부한 내용>
## SSD: KAIST PD Dataset
- Dataset, Dataloader 작성
- Difficult 내용 지우기
- Dataset에 맞게 label map 재정의 {0: background, 1: pedestrian,}
- Checkpoint 로드 후 첫 epoch에서 train 실패
-> VGGBase의 forward 단계인 'out = F.relu(self.conv3_1(out))  # (N, 256, 75, 75)'에서 GPU 용량 초과

# <공부할 내용>
## SSD: KAIST PD Dataset
- Eval 코드 작성
- Testloder를 통해 가져온 GT 박스의 정보를 가져온다.
- gt 박스와 detect박스의 mAP를 계산해 출력
